spk/fengoffice/src/conf_6/privilege

```diff
@@ -0,0 +1,53 @@
+{
+    "defaults": {
+        "run-as": "package"
+    },
```

Comment:
Why not define default "run-as": "root" instead of "package"?
Than you do not need to overwrite all actions with "run-as": "root".

Reply:
Makes sense. Done.